apps/core/templates/components/footer.html

```diff
@@ -1,51 +1,26 @@
+{% load wagtailcore_tags %}
 
 <div class="footer">
     <div class="footer__container">
         <div class="footer__link-blocks">
+          {% for footer_item in footer.footer_items.all %}
             <div class="link-block">
                 <div class="link-block__container">
                     <div class="link-block__icon">
                         {% include "./svg/info.svg" %}
```

Comment:
All of the items now use the same icon, is this intended? If not we could add a `CharField` to the snippet model for the icon and set the `choices` to a predefined `TextChoices` class:

https://docs.djangoproject.com/en/4.1/ref/models/fields/#enumeration-types